plugins/code-climate/api-report.md

```diff
@@ -0,0 +1,108 @@
+## API Report File for "@backstage/plugin-code-climate"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { ApiRef } from '@backstage/core-plugin-api';
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { DiscoveryApi } from '@backstage/core-plugin-api';
+import { Entity } from '@backstage/catalog-model';
+import { IdentityApi } from '@backstage/core-plugin-api';
+import { RouteRef } from '@backstage/core-plugin-api';
+
+// Warning: (ae-missing-release-tag) "CodeClimateApi" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export interface CodeClimateApi {
+  // (undocumented)
+  fetchData(repoID: string): Promise<CodeClimateData | null>;
+}
+
+// Warning: (ae-missing-release-tag) "codeClimateApiRef" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const codeClimateApiRef: ApiRef<CodeClimateApi>;
+
+// Warning: (ae-missing-release-tag) "CodeClimateData" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export type CodeClimateData = {
+  repoID: string;
+  maintainability: {
+    letter: string;
+    value: string;
+  };
+  testCoverage: {
+    letter: string;
+    value: string;
+  };
+  numberOfCodeSmells: number;
+  numberOfDuplication: number;
+  numberOfOtherIssues: number;
+};
+
+// Warning: (ae-missing-release-tag) "codeClimatePlugin" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+const codeClimatePlugin: BackstagePlugin<
+  {
+    root: RouteRef<undefined>;
+  },
+  {}
+>;
+export { codeClimatePlugin };
+export { codeClimatePlugin as plugin };
+
+// Warning: (ae-missing-release-tag) "CodeClimateWidget" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const CodeClimateWidget: ({ entity }: { entity: Entity }) => JSX.Element;
```

Comment:
Avoid exporting the plain components themselves unless needed for some particular reason, just export the extensions